src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -935,17 +938,8 @@ const MeasureGroups = (props: MeasureGroupProps) => {
                 </div>
 
                 {/* Populations Tab */}
-
                 {activeTab === "populations" && (
                   // to do: Condense poppulations Tab into a single component
```

Comment:
Is this to do still valid?

Reply:
Yes.
The only thing I did was decouple things to avoid prop drilling